common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1804,3 +1810,53 @@ def ensure_indexes(self):
 
         # To allow prioritizing draft vs published material
         self.collection.create_index('_id.revision')
+
+    # Some overrides that still need to be implemented by subclasses
+    def convert_to_draft(self, location, user_id):
+        raise NotImplementedError(
+            "{function_name} needs to be implemented in subclass".format(
+                function_name=sys._getframe().f_code.co_name
```

Comment:
Whoa, what's all this magic doing? The message that you're passing to `NotImplementedError` isn't adding any value, because a `NotImplementedError` implies that the method needs to be implemented. You're better off just raising the error without a message, since you don't have anything to add.
```python
def convert_to_draft(self, location, user_id):
    raise NotImplementedError()
```